libs/gltfio/src/AssetLoader.cpp

```diff
@@ -572,6 +566,36 @@ void FAssetLoader::createRenderable(const cgltf_data* srcAsset, const cgltf_node
     }
 }
 
+void FAssetLoader::createMaterialVariants(const cgltf_data* srcAsset, const cgltf_mesh* mesh,
+        Entity entity, FFilamentInstance* instance) {
+    UvMap uvmap {};
+    for (cgltf_size prim = 0, n = mesh->primitives_count; prim < n; ++prim) {
+        const cgltf_primitive* srcPrim = &mesh->primitives[prim];
+        for (size_t i = 0, n = srcPrim->mappings_count; i < n; i++) {
```

Comment:
Use a different loop variable other than `n` for clarity?

Reply:
Good catch, fixed.